Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -73,30 +71,19 @@ NCA = pk_data %>%
             Cmax_0_24     = max(LIDV[NOMTIME>0 & NOMTIME<= 24]), 
             AUC_tau = caTools::trapz(TIME[NOMTIME > 120 & NOMTIME <= 144], LIDV[NOMTIME > 120 & NOMTIME <= 144]),
             Cmax_tau     = max(LIDV[NOMTIME > 120 & NOMTIME <= 144]), 
-            AUC_acc = AUC_tau / AUC_0_24, 
-            Cmax_acc = Cmax_tau / Cmax_0_24,
             SEX      = SEX[1], #this part just keeps the SEX and WEIGHTB covariates
             WEIGHTB  = WEIGHTB[1]) %>%
   gather(PARAM, VALUE,-c(ID, DOSE, SEX, WEIGHTB)) %>%
   ungroup() %>%
-  mutate(VALUE_NORM = VALUE/DOSE)
-
-#AUC dataset for AUC_0_24 and AUCtau
-AUC_0_24 <- NCA[NCA$PARAM=="AUC_0_24",]
-AUC_0_24 <- AUC_0_24[c("VALUE","ID")]
-names(AUC_0_24) <- c("AUC","ID")
-AUC_0_24$PROFDAY = 1
-
-AUCtau <- NCA[NCA$PARAM=="AUC_tau",]
-AUCtau <- AUCtau[c("VALUE","ID")]
-names(AUCtau) <- c("AUC","ID")
-AUCtau$PROFDAY = 6
-if(length(AUCtau[is.na(AUCtau$AUC),]$AUC) > 0 ){
-AUCtau[is.na(AUCtau$AUC),]$AUC<-0
-}
-AUC <- rbind(AUC_0_24, AUCtau)
-AUC$ID <- as.numeric(as.character(AUC$ID))
-AUC <- AUC[order(AUC$ID,AUC$PROFDAY),]
+  mutate(VALUE_NORM = VALUE/DOSE,
+         PROFDAY= ifelse(PARAM == "AUC_0_24" | PARAM == "Cmax_0_24", 1, 6))
+
+#add response data at day 1 and 6 to NCA for additional plots
+RESPONSE <- pd_data[pd_data$PROFDAY%in%c(1,6),
+                    c("ID","PROFDAY","LIDV","TRTACT_low2high")]
+names(RESPONSE) <- c("ID","PROFDAY","Response","TRTACT_low2high")
+NCA <- merge(NCA, RESPONSE, by = c("ID","PROFDAY"))
+NCA$DAY_label <- paste("Day", NCA$PROFDAY)
```

Comment:
Add these lines from PKPD binary:

#create wide pkpd dataset for plotting PK vs PD
pkpd_data_wide = pd_data %>%
  select(ID, NOMTIME, RESPONSE = LIDV) %>%
  right_join(pk_data) %>%
  rename(CONC = LIDV)